packages/react/src/components/LocalNavigation/components/LocalNavigationItem/LocalNavigationItem.tsx

```diff
@@ -40,8 +46,14 @@ export const LocalNavigationItem = React.forwardRef((props, forwardedRef) => {
   const { buttonProps, isPressed } = useButton(
     {
       ...props,
+      onClick,
```

Comment:
Declaring these values to be outside of `props` is not a material change in the code

Reply:
@p44dyoachim this fixes the onPress warning which we are seeing.